openstack/identity/v3/services/requests.go

```diff
@@ -5,10 +5,52 @@ import (
 	"github.com/gophercloud/gophercloud/pagination"
 )
 
+// CreateOptsBuilder allows extensions to add additional parameters to
+// the Create request.
+type CreateOptsBuilder interface {
+	ToServiceCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts provides options used to create a service.
+type CreateOpts struct {
+	// Type is the type of the service.
+	Type string `json:"type"`
+
+	// Enabled is whether or not the service is enabled.
+	Enabled bool `json:"enabled,omitempty"`
```

Comment:
This should be `*bool`.

It looks like services are enabled by default, so when you pass `Enabled: false`, `omitempty` won't include the field in the body. By using a pointer-to-bool, this causes `omitempty` to only omit the field if it wasn't specified at all.

Perhaps confusingly, `bool` is correct in the result struct.

Reply:
Ah yes, I originally made the same mistake on update, but noticed it there. Thanks. Rolled updated PRs.